src/test/java/com/odysseusinc/arachne/datanode/service/DataNodeServiceStandaloneTest.java

```diff
@@ -0,0 +1,81 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Company: Odysseus Data Services, Inc.
+ * Product Owner/Architecture: Gregory Klebanov
+ * Authors: Pavel Grafkin, Vitaly Koulakov, Anastasiia Klochkova, Sergej Suvorov, Anton Stepanov
+ * Created: Oct 16, 2019
+ *
+ */
+
+package com.odysseusinc.arachne.datanode.service;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+
+import com.github.springtestdbunit.DbUnitTestExecutionListener;
+import com.github.springtestdbunit.TransactionDbUnitTestExecutionListener;
+import com.github.springtestdbunit.annotation.DatabaseOperation;
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+import com.github.springtestdbunit.annotation.DatabaseTearDown;
+import com.github.springtestdbunit.annotation.DbUnitConfiguration;
+import com.odysseusinc.arachne.datanode.model.datanode.DataNode;
+import com.odysseusinc.arachne.datanode.model.user.User;
+import javax.ws.rs.NotFoundException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(properties = "datanode.runMode=STANDALONE")
+@ActiveProfiles("test")
+@TestExecutionListeners({ DependencyInjectionTestExecutionListener.class,
+        TransactionDbUnitTestExecutionListener.class,
+        DbUnitTestExecutionListener.class })
+@DbUnitConfiguration(databaseConnection = "primaryDataSource")
+@DatabaseTearDown(value = "/data/dataset/empty-datanode.xml", type = DatabaseOperation.DELETE_ALL)
+public class DataNodeServiceStandaloneTest {
+    @Autowired
+    private DataNodeService dataNodeService;
+    @Autowired
+    private UserService userService;
+    @Autowired
+    private CentralIntegrationService centralIntegrationService;
+
+    @Test
+    @DatabaseSetup(value = "/data/dataset/empty-datanode.xml", type = DatabaseOperation.DELETE_ALL)
+    @DatabaseSetup(value = "/data/dataset/users.xml", type = DatabaseOperation.CLEAN_INSERT)
+    public void createSuccessStandaloneMode() {
```

Comment:
Cannot run the test:
```
2019-10-16 11:40:22.691  INFO 11160 --- [           main] c.o.a.d.s.DataNodeServiceStandaloneTest  : The following profiles are active: test
2019-10-16 11:40:22.705  INFO 11160 --- [           main] o.s.w.c.s.GenericWebApplicationContext   : Refreshing org.springframework.web.context.support.GenericWebApplicationContext@627d8516: startup date [Wed Oct 16 11:40:22 EDT 2019]; parent: org.springframework.context.annotation.AnnotationConfigApplicationContext@19fb8826
2019-10-16 11:40:24.959  INFO 11160 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=333e454c-af3c-3801-be4e-0a3fc20d11c3
2019-10-16 11:40:24.995  WARN 11160 --- [           main] o.s.c.a.ConfigurationClassEnhancer       : @Bean method SpringPropertiesConfig.propertySourcesPlaceholderConfigurer is non-static and returns an object assignable to Spring's BeanFactoryPostProcessor interface. This will result in a failure to process annotations such as @Autowired, @Resource and @PostConstruct within the method's declaring @Configuration class. Add the 'static' modifier to this method to avoid these container lifecycle issues; see @Bean javadoc for complete details.
2019-10-16 11:40:25.000  INFO 11160 --- [           main] EncryptablePropertyResolverConfiguration : String Encryptor custom Bean not found with name 'jasyptStringEncryptor'. Initializing String Encryptor based on properties with name 'jasyptStringEncryptor'
Exception in thread "Thread-2" java.lang.AssertionError
    at com.amazon.support.exceptions.ErrorException.getMessage(Unknown Source)
    at com.amazon.redshift.client.InboundDataHandler.read(InboundDataHandler.java:700)
    at com.amazon.support.channels.AbstractSocketChannel.readCallback(Unknown Source)
    at com.amazon.support.channels.PlainSocketChannel.read(Unknown Source)
    at com.amazon.jdbc.communications.InboundMessagesThread.run(InboundMessagesThread.java:349)
```